scripts/scripts.js

```diff
@@ -155,23 +157,27 @@ export function loadCSS(href) {
 }
 
 /**
- * Adjust all links inside the container to point to current
- * host and not to blog.adobe.com (useful on localhost and authoring host)
- * @param {Element} container The element in which links will be adusted
+ * Turns absolute links within the domain into relative links.
+ * @param {Element} main The container element
  */
-export function adjustLinks(container) {
-  if (container && window.location.host !== 'blog.adobe.com') {
-    container.querySelectorAll('a').forEach((a) => {
+export function makeLinksRelative(main) {
```

Comment:
This needs to get changed in the boilerplate, but this should probably be changed to parent so it's not just assumed it's the `main` element.